cmd/incusd/api_cluster.go

```diff
@@ -3265,6 +3266,12 @@ func evacuateClusterMember(s *state.State, gateway *cluster.Gateway, r *http.Req
 			_ = evacuateClusterSetState(s, nodeName, db.ClusterMemberStateCreated)
 		})
 
+		 // TODO stop all networks here
+		err = networkStopOVN(s)
```

Comment:
So the issue mentioned OVN as that's the biggest issue, but the better approach here is to just stop all networks.

So calling `networkShutdown` to stop all networks.